src/main/java/christmas/domain/event/Event.java

```diff
@@ -0,0 +1,40 @@
+package christmas.domain.event;
+
+import christmas.domain.Amount;
+import christmas.domain.Reservation;
+import christmas.domain.VisitDate;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event {
+
+    private final LocalDate startDate;
+    private final LocalDate endDate;
+
+    private final String name;
+
+    public Event(String start, String end, String name) {
+        this.startDate = parseDate(start);
+        this.endDate = parseDate(end);
+        this.name = name;
+    }
+
+    private static LocalDate parseDate(String date) {
+        return LocalDate.parse(date, DateTimeFormatter.ofPattern("yyyy.MM.dd"));
+    }
+
+    public boolean isEventActive(VisitDate visitDate) {
+        if (visitDate.isAfter(endDate) || visitDate.isBefore(startDate)) {
+            return false;
+        }
+        return true;
+    }
+
+    public Amount process(VisitDate date, Amount amount, Reservation reservation) {
+        return new Amount(0);
+    }
```

Comment:
```suggestion
    public abstract Amount process(VisitDate date, Amount amount, Reservation reservation);
```
동시에 이 함수도 abstract 메소드로 바꿔주면 의미 없는 구현을 피할 수 있을 것 같아요

Reply:
추상클래스는 잘 다뤄보지 않아서 활용하지 못했네요 감사합니다